cloud/pkg/taskmanager/nodeupgradecontroller/node_upgrade_controller.go

```diff
@@ -47,6 +41,16 @@ import (
 	"github.com/kubeedge/kubeedge/cloud/pkg/taskmanager/util/manager"
 	commontypes "github.com/kubeedge/kubeedge/common/types"
 	"github.com/kubeedge/kubeedge/pkg/util/fsm"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	apimachineryType "k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/klog/v2"
+	"oras.land/oras-go/v2/registry"
+	"oras.land/oras-go/v2/registry/remote"
+	"oras.land/oras-go/v2/registry/remote/auth"
+	"oras.land/oras-go/v2/registry/remote/retry"
```

Comment:
Need sort the import. Packages starting with `github.com/kubeedge` should be placed in the third part